packages/ckeditor5-package-generator/lib/templates/ts/sample/ckeditor.ts

```diff
@@ -0,0 +1,112 @@
+declare global {
+	interface Window {
+		editor: ClassicEditor;
+	}
+}
+
+import ClassicEditor from '@ckeditor/ckeditor5-editor-classic/src/classiceditor';
+import CKEditorInspector from '@ckeditor/ckeditor5-inspector';
+
+import Essentials from '@ckeditor/ckeditor5-essentials/src/essentials';
+import Autoformat from '@ckeditor/ckeditor5-autoformat/src/autoformat';
+import BlockQuote from '@ckeditor/ckeditor5-block-quote/src/blockquote';
+import Bold from '@ckeditor/ckeditor5-basic-styles/src/bold';
+import Heading from '@ckeditor/ckeditor5-heading/src/heading';
+import Image from '@ckeditor/ckeditor5-image/src/image';
+import ImageCaption from '@ckeditor/ckeditor5-image/src/imagecaption';
+import ImageStyle from '@ckeditor/ckeditor5-image/src/imagestyle';
+import ImageToolbar from '@ckeditor/ckeditor5-image/src/imagetoolbar';
+import ImageUpload from '@ckeditor/ckeditor5-image/src/imageupload';
+import Base64UploadAdapter from '@ckeditor/ckeditor5-upload/src/adapters/base64uploadadapter';
+import Indent from '@ckeditor/ckeditor5-indent/src/indent';
+import Italic from '@ckeditor/ckeditor5-basic-styles/src/italic';
+import Link from '@ckeditor/ckeditor5-link/src/link';
+import List from '@ckeditor/ckeditor5-list/src/list';
+import MediaEmbed from '@ckeditor/ckeditor5-media-embed/src/mediaembed';
+import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
+import Table from '@ckeditor/ckeditor5-table/src/table';
+import TableToolbar from '@ckeditor/ckeditor5-table/src/tabletoolbar';
+import CodeBlock from '@ckeditor/ckeditor5-code-block/src/codeblock';
+import Code from '@ckeditor/ckeditor5-basic-styles/src/code';
+
+import MyPlugin from '../src/myplugin';
+
+/* global document, window */
+
+ClassicEditor
+	.create( document.getElementById( 'editor' )!, {
+		plugins: [
+			// `MyPlugin` is an example plugin generated by the package generator.
+			MyPlugin,
```

Comment:
I've got the following error here:

![Zrzut ekranu 2022-08-02 125635](https://user-images.githubusercontent.com/56868128/182360902-23d3e28b-ffa4-454d-a3da-11ac94e72ecf.png)

It looks like it is sufficient to include the `sample` folder in `tsconfig.json` in the `"include"` array:
```js
"include": [
	"sample",    // added
	"src",
	"tests",
	"typings"
]
```